definitions.py

```diff
@@ -11,4 +11,36 @@ class EncounterType(StrEnum):
     ELITE = auto()
     REST_SITE = auto()
     BOSS = auto()
-    UNKNOWN = auto()
\ No newline at end of file
+    SHOP = auto()
+    UNKNOWN = auto()
+
+class Rarity(StrEnum):
+    BASIC ='Basic'
+    BOSS ='Boss'
+    COMMON ='Common'
+    CURSE ='Curse'
+    EVENT ='Event'
+    RARE ='Rare'
+    SHOP ='Shop'
+    SPECIAL ='Special'
+    STARTER ='Starter'
+    UNCOMMON ='Uncommon'
+
+class CardType(StrEnum):
+    ATTACK = 'Attack'
+    SKILL = 'Skill'
+    POWER = 'Power'
+    STATUS = 'Status'
+    CURSE = 'Curse'
+    ENTROPIC = 'Entropic'
```

Comment:
The types that are defined for the potions were not thought through at all. Those types that you've defined are most likely not the best way of doing it.

Reply:
Like those were made a while ago and i didn't really know what that was supposed to do anyways.

Reply:
You're talking about Entropic, Snecko, and Dark Essence? I can remove those. Was just trying to be consistent with what's there.

Reply:
Ok.